eng/pipelines/runtime-linker-tests.yml

```diff
@@ -0,0 +1,33 @@
+# The purpose of this pipeline is to exercise local developer workflow in the consolidated
+# runtime repo. In particular, it is supposed to run the root "build" script just like any
+# normal developer normally would and monitor regressions w.r.t. this fundamental scenario.
+
+trigger: none
+
+pr:
+  branches:
+    include:
```

Comment:
Add release/* so that it runs in future release branches.

Reply:
sounds good.